plexapi/media.py

```diff
@@ -124,7 +124,41 @@ def audioStreams(self):
     def subtitleStreams(self):
         """ Returns a list of :class:`~plexapi.media.SubtitleStream` objects in this MediaPart. """
         return [stream for stream in self.streams if stream.streamType == SubtitleStream.STREAMTYPE]
-
+        
+    def setDefaultAudioStream(self, stream):
+        """ Set the default :class:`~plexapi.media.AudioStream` for this MediaPart.
+
+            Parameters:
+                stream (:class:`~plexapi.media.AudioStream`): AudioStream to set as default 
+            
+            Raises:
+                :class:`plexapi.exceptions.BadRequest`: If stream is not an AudioStream.
+        """
+        if type(stream) == AudioStream:
+            key = "/library/parts/%d?audioStreamID=%d&allParts=1" % (self.id, stream.id)
+            self._server.query(key, method=self._server._session.put)
+        else:
+            raise BadRequest("Object 'stream' is not an AudioStream.")
+            
+    def setDefaultSubtitleStream(self, stream):
+        """ Set the default :class:`~plexapi.media.SubtitleStream` for this MediaPart.
+            
+            Parameters:
+                stream (:class:`~plexapi.media.SubtitleStream`): SubtitleStream to set as default.
+            
+            Raises:
+                :class:`plexapi.exceptions.BadRequest`: If stream is not a SubtitleStream.
+        """
+        if type(stream) == SubtitleStream:
+            key = "/library/parts/%d?subtitleStreamID=%d&allParts=1" % (self.id, stream.id)
+            self._server.query(key, method=self._server._session.put)
+        else:
+            raise BadRequest("Object 'stream' is not a SubtitleStream.")
+        
+    def resetSubtitles(self):
```

Comment:
Can you please rename the method a little? Currently we have `setDefaultSubtitleStream` and `resetSubtitles`, which looks a bit strange — I believe that the second one should be `resetDefaultSubtitleStream`